core/quantity/src/Formatter/FormatterSpec.ts

```diff
@@ -100,7 +104,23 @@ export class FormatterSpec {
    */
   public static async create(name: string, format: Format, unitsProvider: UnitsProvider, inputUnit?: UnitProps): Promise<FormatterSpec> {
     const conversions: UnitConversionSpec[] = await FormatterSpec.getUnitConversions(format, unitsProvider, inputUnit);
-    return new FormatterSpec(name, format, conversions, inputUnit);
+    const spec = new FormatterSpec(name, format, conversions, inputUnit);
```

Comment:
it seems wrong that you have to do extra to the FormatterSpec after creating it even though the information is being passed into the constructor.

Reply:
Fixed